resource/menu/client/cl_player_ids.lua

```diff
@@ -117,6 +117,55 @@ end)
 
 RegisterCommand('txAdmin:menu:togglePlayerIDs', togglePlayerIDsHandler)
 
+local playersBlips = {}
+---Refreshes blips on client
+---@param blipsData {[string]: {coords: {x:number,y:number,z:number,h:number}, name:string}}
+local function refreshPlayerBlips(blipsData)
+    for k, v in pairs(playersBlips) do
+        RemoveBlip(v)
+      end
+      playersBlips = {}
+      for playerId, v in pairs(blipsData) do
+        local blip = AddBlipForCoord(v.coords.x+0.01, v.coords.y+0.01, v.coords.z+0.01)
```

Comment:
i think use [AddBlipForEntity](https://docs.fivem.net/natives/?_0x30822554) native because blip directly added to player ped. 
in the case of onesync, it may cause problems for remote players, the current solution may remain there.
_Write what you think._

Reply:
Yea, if onesync would be on, adding blips for each player ped entity (not coord) would result lots of blips on ped controlled by player (-1) instead of other players peds.

Reply:
There won't be many blips, but you won't be able to retrieve peds from distant players.
```lua
local playerPed = GetPlayerPed(serverId)
if not DoesEntityExist(playerPed) then --ped invalid
--attach to coords
end
```

Reply:
This will always return true tho, as it if the player is far away it would be the same as GetPlayerPed(-1)
```lua
if not DoesEntityExist(playerPed) then --ped invalid
```

This would do tho, but I do not think it is neccessary really 🤔 
```lua
local playerPed = GetPlayerPed(serverId)
if playerPed  ~= PlayerPedId() then --ped invalid
--attach to coords
else
--attach to entity
end
```

Reply:
I thought onesync doesn't work that badly 😢

Reply:
Couldn't you use [`NetToPed(playerID)](https://docs.fivem.net/natives/?_0xBDCD95FC216A8B3E)?